src/services/event.js

```diff
@@ -273,11 +277,83 @@ const makeMatchesFilters = (filters) => (event) => {
   return false;
 };
 
+/**
+ * Splits up multiple occurrences of events into individual events
+ * and re-sorts the array
+ *
+ * @param {Object[]} events the array of events to process
+ * @returns {Object[]} the flattened array of events after splitting multiple occurrences
+*/
+const processMultipleOccurences = (events) => {
+
+  var splitEvents = [];
+
+  for (let i = 0; i < events.length; i++) {
+
+    let thisEvent = events[i];
```

Comment:
You can use a `for...of` loop here for more concise syntax:
```suggestion
  for (const event of events) {
```